src/main/java/com/ita/if103java/ims/controller/SavedItemController.java

```diff
@@ -0,0 +1,72 @@
+package com.ita.if103java.ims.controller;
+
+import com.ita.if103java.ims.dto.*;
+import com.ita.if103java.ims.entity.Associate;
+import com.ita.if103java.ims.entity.User;
+import com.ita.if103java.ims.service.ItemService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/savedItems")
+public class SavedItemController {
+    private ItemService itemService;
```

Comment:
For all this methods  need to use accountId. 
You can use @AuthenticationPrincipal User user in this methods.

**IMPORTANT: In order to not block your PR please do this in next one!**